src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/debugImageDetails/utils.tsx

```diff
@@ -1 +1,2 @@
 export const INTERNAL_SOURCE = 'sentry:project';
+export const INTERNAL_SOURCE_LOCATION = 'sentry://project_debug_file/';
```

Comment:
Is this being properly encoded in the url?

Reply:
I am not using this const in any request.

Reply:
That's my bad then, I probably misunderstood what this does.